src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -596,3 +600,33 @@ def closure_to_tasklet_sdfg(
     for output in outputs:
         context.body.arrays[output.value.data].transient = False
     return context, outputs
+
+
+def lambda_to_tasklet_sdfg(
+    node: itir.StencilClosure,
+    offset_provider: dict[str, Any],
+    node_types: dict[int, next_typing.Type],
+) -> tuple[Context, Sequence[tuple[str, ValueExpr]], list[ValueExpr]]:
+    body = dace.SDFG("tasklet_toplevel")
+    state = body.add_state("tasklet_toplevel_entry")
+    symbol_map: dict[str, ValueExpr | IteratorExpr] = {}
+
+    context = Context(body, state, symbol_map)
+
+    def get_lambda() -> Lambda:
+        stencil_fobj = cast(FunCall, node.stencil)
+        return cast(Lambda, stencil_fobj.args[0])
+
+    call = itir.Lambda(expr=get_lambda().expr, params=get_lambda().params)
+    translator = PythonTaskletCodegen(offset_provider, context, node_types)
+
+    # add lambda parameters to symbol map
+    for p in get_lambda().params:
+        pnode = state.add_access(p.id)
+        symbol_map[p.id] = ValueExpr(value=pnode.data, dtype=dace.float64)
```

Comment:
@petiaccja @tehrengruber Here is where I create the `ValueExpr` nodes. I agree, this could be a problem. In particular, see the hard-coding of float64 type, which obviously is a quick and dirty solution. Can you suggest a better way?

Reply:
It is possible to retrieve the data type for the lambda input arguments:
```suggestion
        pnode_type = node_types[id(p)]
        symbol_map[p.id] = ValueExpr(value=pnode.data, dtype=itir_type_as_dace_type(pnode_type.dtype))
```

Reply:
We have found a better solution, this code was removed.